examples/js/libs/inflate.min.js.map

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Yes, it works, well I guess, Ill just remove that line then and  update the code.. :)